openpdf/src/main/java/com/lowagie/text/pdf/PdfDocument.java

```diff
@@ -256,17 +256,17 @@ void addkey(String key, String value) {
          * @param documentFileAttachment	the attached files
          * @param writer the writer the catalog applies to
          */
-        void addNames(TreeMap localDestinations, HashMap documentLevelJS, HashMap documentFileAttachment, PdfWriter writer) {
+        void addNames(TreeMap<String, Object[]> localDestinations, HashMap<String, PdfIndirectReference> documentLevelJS, HashMap<String, PdfIndirectReference> documentFileAttachment, PdfWriter writer) {
```

Comment:
I would be also nice, if it is simple, to have return values and parameters of methods use the interface of collections, if the implementation doesn't really care if the map is a HashMap or a TreeMap. So many this could be preferable:

void addNames(Map<String, Object[]> localDestinations, Map<String, PdfIndirectReference> documentLevelJS, Map<String, PdfIndirectReference> documentFileAttachment, PdfWriter writer) .

(you don't need to change this now, just a hint for future Pull Requests. This will be merged, anyway.)